### ISSUE TITLE
Caverne à Bijoux

### ISSUE THREAD

**Issue by Keyzu** [Keyzu/Pierre-Kiroule#6]
Mobs/Quêtes qui loot des rubijou brut, à échanger contre des Rubijou (poli ^^').
Rubijou -> Monnaie pour acheter des anneaux/bijoux/amulettes

Caverne à Bijoux -> [HYPOTHESE] Vagues de monstres qui te fonce dessus, une quête avec timer pour en buter le plus possible, et par moment des vagues avec seulement un boss.

**Comment by ghost** [comment]
Yes ! J'aimais bien aussi le système des cristaux à cliquer pour être tp vers une sorte de faille un peu à la diablo si vous connaissez ;)

En tous cas,Hips va devoir s'amuser avec les tables des loots ! :D @

**Comment by Keyzu** [comment]
Oui c'est vrai ce concept a l'air sympa aussi, style faille nephalem j'aime bien :p Après il faut voir si c'est du solo/multi à l'arrache (tp dans une zone ouverte) / instance.
Yep il y a du boulot avec les tables de loot !

**Comment by ghost** [comment]
Ça pourrait être du multi en zone ouverte, surpeuplée :)

**Comment by ghost** [comment]
Pour le coup je pense qu'il serait intéressant d'ajouter des quêtes journalières pour que les joueurs ne puissent pas farm en illimiter les mobs/boss/quêtes. Pour les boss on peut changer la table de loot des mobs rares du jeu. Ceux-ci n'étant pas forcément facile à trouver ça peut-être intéressant je pense.
